### PR TITLE
Fixes debian

### PR DESCRIPTION
**Pull request by a-skr**
#### Description:

- update debian12 profiles
- update a chronyd rule to handle debian /etc sub-directories organization.
- fix a regex in sysctl template

### CODE REVIEW

File: linux_os/guide/services/ntp/chronyd_specify_remote_server/oval/shared.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
   <ind:textfilecontent54_object comment="Ensure at least one NTP server is set"
   id="object_chronyd_remote_server" version="1">
-    <ind:filepath operation="pattern match">^/etc/chrony\.(conf|d/.+\.conf)$</ind:filepath>
+    <ind:filepath operation="pattern match">^/etc/(?:chrony/)?chrony\.(conf|d/.+\.conf)$</ind:filepath>
```
Comment (jan-cerny) [inline linux_os/guide/services/ntp/chronyd_specify_remote_server/oval/shared.xml:18]: This needs to be reworked to use product properties `chrony_conf_path` and `chrony_d_path` correctly.

Comment (a-skr) [reply linux_os/guide/services/ntp/chronyd_specify_remote_server/oval/shared.xml]: commit has been amended accordingly.

File: products/debian12/product.yml [hunk 2]
```diff
@@ -35,3 +35,5 @@ platform_package_overrides:
   pam: libpam-runtime
   shadow: login
   sssd: sssd-common
+  chrony_conf_path: "/etc/chrony/chrony.conf"
+  chrony_d_path: "/etc/chrony/chrony.d"
```
Comment (jan-cerny) [inline products/debian12/product.yml:39]: the CI fails because you will have to update the product stability data

Comment (a-skr) [reply products/debian12/product.yml]: I have updated the product stability data (and rebased onto current master).

**Review by Mab879 — CHANGES_REQUESTED** [review]
Thanks for the PR, I have one comment.

File: linux_os/guide/services/ntp/chronyd_specify_remote_server/oval/shared.xml [hunk 3]
```diff
@@ -15,7 +15,8 @@
 
   <ind:textfilecontent54_object comment="Ensure at least one NTP server is set"
   id="object_chronyd_remote_server" version="1">
-    <ind:filepath operation="pattern match">^/etc/chrony\.(conf|d/.+\.conf)$</ind:filepath>
+    <!-- ind:filepath operation="pattern match">^/etc/(?:chrony/)?chrony\.(conf|d/.+\.conf)$</ind:filepath -->
```
[the remark this answers is not in the document]
Comment (a-skr) [reply linux_os/guide/services/ntp/chronyd_specify_remote_server/oval/shared.xml]: Sorry, I did miss this message from two weeks ago. I just updated my branch.

**Review by dodys — APPROVED** [review]
lgtm, thanks!

**Review by Mab879 — APPROVED** [review]

**Comment by Mab879** [comment]
Automatus passes locally

```
$ ./automatus.py rule --datastream ../build/ssg-rhel9-ds.xml --remediate-using bash  --libvirt qemu:///system automatus_rhel9_4 chronyd_specify_remote_server 
Setting console output to log level INFO
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/mburket/Developer/ComplianceAsCode/content/tests/logs/rule-custom-2024-08-02-1247/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_chronyd_specify_remote_server
INFO - Script correct.pass.sh using profile (all) OK
INFO - Script correct_pool.pass.sh using profile (all) OK
INFO - Script file_empty.fail.sh using profile (all) OK
INFO - Script file_missing.fail.sh using profile (all) OK
INFO - Script line_missing.fail.sh using profile (all) OK
INFO - Script multiple_servers.pass.sh using profile (all) OK
INFO - Script server_not_specified.fail.sh using profile (all) OK
```

**Pull request merged**